src/Sulu/Component/Content/Types/Rlp/Strategy/RlpStrategy.php

```diff
@@ -240,6 +236,7 @@ private function adaptResourceLocators(
                 // write value to node
                 $translatedProperty = $this->nodeHelper->getTranslatedProperty($property, $languageCode);
                 $contentType->write($node, $translatedProperty, $userId, $webspaceKey, $languageCode, $segmentKey);
+                $this->save($node, $translatedProperty->getValue(), $userId, $webspaceKey, $languageCode, $segmentKey);
```

Comment:
Has to be added, because this logic is not part of the content type anymore. Also combined the `save` method with `move`, so that the check which one has to be called is not spread all over the codebase.